scenario_player/runner.py

```diff
@@ -153,289 +335,326 @@ def local_seed(self) -> str:
             seed = seed_file.read_text().strip()
         return seed
 
-    def select_chain(self, chain_urls: Dict[str, List[str]]) -> Tuple[str, List[str]]:
-        """Select a chain and return its name and RPC URL.
+    def ensure_token_network_discovery(
```

Comment:
Same as https://github.com/raiden-network/scenario-player/blob/edd041e816776c141a1182f089ab6bb75a976e91/scenario_player/runner.py#L228 with an small improvement, now it not only checks there is a token network, but also that all have the same one